bentoml/_internal/io_descriptors/image.py

```diff
@@ -140,7 +141,9 @@ async def predict_image(f: Image) -> NDArray[Any]:
     def __init__(
         self,
         pilmode: _Mode | None = DEFAULT_PIL_MODE,
-        mime_type: str = "image/jpeg",
+        mime_type: str | t.Iterable[str] | None = None,
+        *,
+        accept_all_images: bool = True,
```

Comment:
Maybe `restrict_allowed_input_types` or something along those lines instead?

Reply:
`allow_all_mime_types`?

Reply:
This is for compatibility, because we specifically allowed only `image/*` before.

Reply:
`restrict_allowed_input_types` make sense.